starfish/pipeline/filter/gaussian_low_pass.py

```diff
@@ -0,0 +1,58 @@
+from skimage.filters import gaussian
+import numpy
+
+from starfish.munge import swap
+from ._base import FilterAlgorithmBase
+
+
+# TODO: team, decide on API for function calls
+# the starfish import is a bit kludgy because it means we need to:
+# starfish.GaussianFilter.filter() (probably just an import change)
+class GaussianLowPass(FilterAlgorithmBase):
+    @classmethod
+    def from_cli_args(cls, args):
+        return GaussianLowPass()
+
+    @classmethod
+    def get_algorithm_name(cls):
+        return "gaussian_low_pass"
+
+    @classmethod
+    def add_arguments(cls, group_parser):
+        group_parser.add_argument(
+            "--sigma", default=1, type=int, help="size of morphological masking disk")
+
+    @classmethod
+    def filter(cls, image, sigma, *args, **kwargs):
+        """Applies a gaussian low pass filter to an image. Implementation simply calls skimage.filters.gaussian
+
+        Parameters
+        ----------
+        image : np.ndarray
+            input image to be filtered
+        sigma : int
+            standard deviation of gaussian kernel
+        kwargs : dict
+            Not used. Enables passing of arbitrary arguments from the CLI.
+
+        Returns
+        -------
+        blurred : np.ndarray
```

Comment:
what's blurred?

Reply:
blurred is the output, of type np.ndarray. (gaussian low pass blurs an image).

Reply:
I think this is kind of weird.  It's not like the return value is named 'blurred'.  I would almost prefer that it be a textual explanation for what is being returned.